docs_src/microservices/device/supported/device-usb-camera/supplementary-info/advanced-options.md

```diff
@@ -38,6 +38,104 @@ To configure the username and password for rtsp authentication when building you
     RUN sed -i 's,externalAuthenticationURL:,externalAuthenticationURL: http://localhost:8000/rtspauth,g' rtsp-simple-server.yml
     ```
 
+## Set Device Parameters
+### Set frame rate
+This option sets the frame rate for the capture device.
+
+1. Execute the `DataFormat` api call to see the available framerates:
+```bash
+curl http://localhost:59882/api/v3/device/name/<device name>/DataFormat?PathIndex=<device path>
```

Comment:
You probably should also let the user know what is this device path and give an example on how to use it.